recipes/sdf/all/test_package/test_package.c

```diff
@@ -8,30 +8,8 @@
 
 int main(int argc, char **argv)
 {
-    if (argc < 2) {
-        fprintf(stderr, "Need at least one argument\n");
-        return 1;
-    }
-
     int width, height, bpp;
-    unsigned char* img_data = stbi_load(argv[1], &width, &height, &bpp, 0);
-    if(img_data == NULL)
-    {
-        fprintf(stderr, "Could not load image: %s\n", stbi_failure_reason());
-        return 1;
-    }
-
-    unsigned char* dest_data = malloc(width * height * bpp);
-    if(dest_data == NULL)
-    {
-        stbi_image_free(img_data);
-        return 1;
-    }
-
-    sdfBuildDistanceField(dest_data, width, 2.0f, img_data, width, height, width);
-
-    free(dest_data);
-    stbi_image_free(img_data);
-
+    unsigned char* img_data = stbi_load("not a valid/path/fake_img.png", &width, &height, &bpp, 0);
+    printf("img_data: %p\n", img_data);
```

Comment:
It's probably UB to print a `NULL` pointer in this case, (https://stackoverflow.com/a/44996472) I'd keep the error message with the check in there, but make it return ok